components/questions.js

```diff
@@ -12,7 +13,7 @@ export default function Questions(props) {
   const [answer, setAnswer] = useState(null);
   const [confidence, setConfidence] = useState(DEFAULT_CONFIDENCE);
   const [comment, setComment] = useState("");
-  const [error, setError] = useState(false);
```

Comment:
For the purposes of your changes, you don't need to change the initial value, as you will evaluate below if error is true or false, so having it `false` to start with is a good choice.